core/src/main/kotlin/fr/sncf/osrd/stdcm/graph/STDCMPathfinding.kt

```diff
@@ -141,6 +142,16 @@ class STDCMPathfinding(
                     graph.tag,
                     temporarySpeedLimitManager,
                 ) ?: return null
+        val travelTime = res.envelope.totalTime
+        val stopTime = res.stopResults.sumOf { it.duration }
+        Span.current().setAttribute("departure delay", res.departureTime.toString())
+        Span.current().setAttribute("total travel time", travelTime.toString())
+        Span.current().setAttribute("total stop time", stopTime.toString())
+        Span.current()
+            .setAttribute(
+                "travel time / max travel time",
+                ((travelTime + stopTime) / maxRunTime).toString(),
+            )
         logger.info(
             "departure time = +${res.departureTime.toInt()}s, " +
```

Comment:
To match with (clearer) span attribute:
```suggestion
            "departure delay = +${res.departureTime.toInt()}s, " +
```
▶️ Also better to have all the values logged under the same name than span's attributes.

Reply:
Fixed